image.yaml

```diff
@@ -2,8 +2,8 @@ schema_version: 1
 
 name: "jboss-eap-7-tech-preview/eap-cd-openshift-rhel8"
 description: "The JBoss EAP continuous delivery (JBoss EAP CD) releases are intended to be Technology Preview - The JBoss EAP CD OpenShift container image is provided as technology preview. It is intended for development use only. It should NOT be deployed on production or in environments that are not intended for development use."
-version: "20.0"
-from: "ubi8:8-released"
+version: "21.0"
+from: "registry.redhat.io/ubi8/ubi:latest"
```

Comment:
@luck3y , why do we need to change the image name?

Reply:
:8-released is no longer provided in brew, and just using :latest could risk picking up an image that hasn't been QE'd yet -- so the safest option is to use the full registry name here.